good_first_issues/utils/__init__.py

```diff
@@ -104,6 +104,16 @@ def identify_limit(limit: Optional[int], all: bool) -> Optional[int]:
         return 10
 
 
+def get_row_ids(issue_count, limiter):
```

Comment:
@jungrishi  I missed it on the earlier review, can you also add type annotation for the arguments and return type here. 

```python
def get_row_ids(issue_count: int, limiter: Optional[int]) -> Iterable[int]:
```

Also the the index from 1 needs to be implemented on the REST API output as well. 
https://github.com/yankeexe/good-first-issues/blob/2d7941c72d49e0b93d3ef217e192ecd957109878/good_first_issues/rest/commands.py#L74

Reply:
sure

Reply:
done